core/modules/layout/css/layout.admin.css

```diff
@@ -392,3 +392,7 @@
 [dir="rtl"] .edit-layout-top .dropbutton-wrapper {
   float: left;
 }
+
+.layout-editor-block:focus {
+  box-shadow: 5px 5px 5px 0 #000;
```

Comment:
Just a question: is this the usual design pattern to highlight focused items? It might be invisible on dark admin themes.

Reply:
This focus seems "heavy" to me compared with other focus states. I do think it would be a good idea to give Seven a more substantial focus state (such as a blue or other color highlight), but this is the generic styling (not specific to Seven), which tends to stay within gray colors.

Reply:
This was I suppose a "placeholder" for a focus CSS. I'm open to ideas, or leaving this off if its unnecessary.